### PR TITLE
flux-mini: add `-q, --queue=NAME` option

### LINKED ISSUE #4437: add flux mini --queue option
Problem: the only way to specify a queue when submitting a job is to use the `flux mini --setattr=system.queue=NAME` option.  There should be a shorter option like `--queue=NAME`.

### PR DESCRIPTION
**Pull request by grondo**
The title says it all. This is just shorthand for `--setattr=queue=NAME`.

Fixes #4437

### CODE REVIEW

File: src/cmd/top/joblist_pane.c [hunk 1]
```diff
@@ -29,6 +29,9 @@ struct joblist_pane {
     json_t *jobs;
     struct ucache *ucache;
 
+    const char *queue;
+    bool show_queue;
+
```
Comment (chu11) [inline src/cmd/top/joblist_pane.c:34]: you have this in the ` flux-mini: remove -q short option for --quiet ` commit, I guess a merge conflict.

File: src/cmd/top/joblist_pane.c [hunk 2]
```diff
@@ -62,6 +65,10 @@ static json_t *get_current_job (struct joblist_pane *joblist)
 }
 
 
+static bool show_queue (struct joblist_pane *joblist)
+{
+}
```
Comment (chu11) [inline src/cmd/top/joblist_pane.c:70]: you have this in the ` flux-mini: remove -q short option for --quiet ` commit, I guess a merge conflict.

File: src/cmd/top/joblist_pane.c [hunk 3]
```diff
@@ -243,12 +250,13 @@ void joblist_pane_query (struct joblist_pane *joblist)
...
                              0,
                              0,
-                             "{s:i s:i s:i s:i s:[s,s,s,s,s,s,s]}",
+                             "{s:i s:i s:i s:i s:[s,s,s,s,s,s,s,s]}",
                              "max_entries", win_dim.y_length - 1,
                              "userid", FLUX_USERID_UNKNOWN,
                              "states", FLUX_JOB_STATE_RUNNING,
                              "results", 0,
                              "attrs",
+                               "queue",
```
Comment (chu11) [inline src/cmd/top/joblist_pane.c:259]: you have this in the `flux-mini: remove -q short option for --quiet` commit, I guess a merge conflict.

**Review by chu11 — COMMENTED** [review]
overall looks good, but i think maybe you commited code from a different feature in one of the commits?

File: src/cmd/top/joblist_pane.c (hunk 3 above)
Comment (grondo) [reply src/cmd/top/joblist_pane.c:259]: Yeah accidentally committed some WIP stuff. Oops

**Comment by grondo** [comment]
Ok, I force pushed with the unrelated code from my working directory removed :facepalm:

**Review by chu11 — APPROVED** [review]
ok, LGTM!

**Comment by grondo** [comment]
Thanks! I've set MWP

**Comment by grondo** [comment]
Restarted the `bionic - py3.7` builder due to a recurrence of #4596

**Comment by grondo** [comment]
@mergifyio refresh

**Pull request merged**